inst/LogLogistic/exp_long_functions.stan

```diff
@@ -0,0 +1,101 @@
+row_vector ifelse(int[] condition, row_vector yes, row_vector no) {
```

Comment:
I think it would now be already a good time to make a `utils.stan` file which is appended to the functions when doing the `complete()` call.

Reply:
I will implement it into this branch then

Reply:
that is a good idea I think! Because `as.character()` creates the final stan program for any `StanModule`.

Reply:
You can find my implementation in the new update